warehouse/dbt_project.yml

```diff
@@ -21,9 +21,8 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "target"
   - "dbt_packages"
 
-
-# Configuring models
-# Full documentation: https://docs.getdbt.com/docs/configuring-models
+vars:
+  INCREMENTAL_PARTITIONS_LOOKBACK_DAYS: 30
```

Comment:
did you check if there are any other incremental RT tables that should use this?

Reply:
I don't think so right now, because we don't actually materialize actual RT data right now.